### DISCUSSION TITLE
Errors on console [Mar 05]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5283] [Discord Threads]
> 🙋‍♂️ **Joecode**  
## Has anyone seen this type of error before? I is happening intermittently when I refresh the page or navigate to a view. When this happens the table shows the row numbers only but none of the columns or other table data is there. Any ideas what could be causing this or how to debug?  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1070440045808468171/image.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### we need more info on this. Are you running locally? docker? any actions to trigger these errors? also the project info.  
#  
> 🙋‍♂️ **Joecode**  
#### Hi <@495466997304262658> , sorry I should have included more info. Running a custom docker image based off 0.101.0-beta.0. Here is project info:   
####    
#### Node: **v16.17.1**   
#### Arch: **x64**   
#### Platform: **linux**   
#### Docker: **true**   
#### RootDB: **pg**   
#### PackageVersion: **0.101.0-beta.0**   
####    
#### This is triggered intermittently by using the browser refresh button when I have a view open. I haven't seen any network issues, so I don't believe it's related to a backend api call error. If I just keep refreshing the page I will get this every so often.  
#  
> 🙋‍♂️ **wkw**  
#### actually I've seen similar errors before but not recently. I guess it has been resolved after 0.101.0-beta.0. I'd suggest you to use the latest version  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1070440045980418219)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
